cmd/operator/main.go

```diff
@@ -95,40 +114,48 @@ func main() {
 
 	withWebhook := true
 
-	// TODO(mythi): add a StringVar flag to select which controllers to enable
-	for _, device := range []string{"dsa", "fpga", "gpu", "qat", "sgx"} {
+	for device, enabled := range devices {
+		if !enabled {
+			continue
+		}
 		if err = setupControllerAndWebhook[device](mgr, ns, withWebhook); err != nil {
 			setupLog.Error(err, "unable to initialize controller", "controller", device)
 			os.Exit(1)
 		}
 	}
 
```

Comment:
We should either validate --device parameter or just stop at this point if we haven't called setupContollerAndWebhook at the end of the loop.